.gitattributes

```diff
@@ -1,6 +1,5 @@
+# Default behaviour, for if core.autocrlf isn't set
+* text=auto
+
 *.ml linguist-language=OCaml
 *.mli linguist-language=OCaml
-
-# We are pinning wasm_of_ocaml using this file in the CI. This would
-# fail on Windows otherwise.
-VERSION -text
```

Comment:
This needs to stay for now

Reply:
The latest master includes a fix that makes this no longer needed.

Reply:
In fact, CI is not failing!

Reply:
Changes in the file don't fit with the commit message. I'll merge and amend the message

Reply:
FYI, I'm reverting this change in https://github.com/ocsigen/js_of_ocaml/pull/1789